src/index.js

```diff
@@ -3,6 +3,7 @@ import { CID } from 'multiformats'
 import { extractVerifiedContent } from './utils/car.js'
 import { asAsyncIterable, asyncIteratorToBuffer } from './utils/itr.js'
 import { randomUUID } from './utils/uuid.js'
+import { memoryStorage } from './utils/storage.js'
```

Comment:
can you rename the file to `memory-storage.js`. Let's do 1 module per storage implementation.

Reply:
also, let's create a dir dedicated to storage instances, like `/storage`

Reply:
I added a common index.js to export from and define the jsdoc interface, you cool with that?